consensus-peer/config.proto

```diff
@@ -14,10 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package helper
+syntax = "proto3";
 
-import "testing"
+package main;
 
-func TestHandler(t *testing.T) {
-	t.Skip("Handler functions already tested in other consensus components")
-}
+import "github.com/hyperledger/fabric/consensus/pbft/config.proto";
+
+message config {
+        pbft.batch_config consensus = 1;
+        map<string, bytes> peers = 2;
+};
```

Comment:
I think it's a generally great idea to have a protobuf representation of our config.  It would be nice to include this representation (or maybe a refactored version of it) in the `handshake` message defined in the other proto, to ensure that all peers have the necessary same config.  Maybe separate the config into categories of 'network config' and 'individual config' or something like that, where everyone's 'network config' must match.